src/test/java/carracing/controlstatement/IFTest.java

```diff
@@ -0,0 +1,21 @@
+package carracing.controlstatement;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IFTest {
+	@Test
+	void booleanValue_true_true() {
+		Boolean b = new IF(new Boolean.SmartTrue());
```

Comment:
상위 클래스를 직접 호출하지 않아도 될 것 같아요 
```suggestion
		Boolean b = new IF(new SmartTrue());
```